File: data/code_pairs/fortran/1449.f90
module sizes
  implicit none
  private

  integer, parameter, public :: Ndim=3
  integer, dimension(Ndim), parameter, public :: mesh=(/ 100, 100, 100 /)
  integer, parameter, public :: meshx=mesh(1)
  integer, parameter, public :: meshy=mesh(2)
  integer, parameter, public :: meshz=mesh(3)
end module sizes

File: data/code_pairs/cpp/1449.cpp
#ifndef SIZES_H
#define SIZES_H

#include <array>

namespace sizes {
    constexpr int Ndim = 3;
    constexpr std::array<int, Ndim> mesh = {100, 100, 100};
    constexpr int meshx = mesh[0];
    constexpr int meshy = mesh[1];
    constexpr int meshz = mesh[2];
}

#endif // SIZES_H